akka-actor-typed-tests/src/test/java/jdocs/akka/typed/FSMDocTest.java

```diff
@@ -14,182 +14,176 @@
 import java.util.LinkedList;
 import java.util.List;
 
-public class FSMDocTest {
+interface FSMDocTest {
 
+  // #simple-state
   // #simple-events
-  interface Event {}
-  // #simple-events
+  public abstract class Buncher {
+    // #simple-state
 
-  public
-  // #simple-events
-  static final class SetTarget implements Event {
-    private final ActorRef<Batch> ref;
+    public interface Event {}
 
-    public SetTarget(ActorRef<Batch> ref) {
-      this.ref = ref;
-    }
+    public static final class SetTarget implements Event {
+      private final ActorRef<Batch> ref;
+
+      public SetTarget(ActorRef<Batch> ref) {
+        this.ref = ref;
+      }
 
-    public ActorRef<Batch> getRef() {
-      return ref;
+      public ActorRef<Batch> getRef() {
+        return ref;
+      }
```

Comment:
Do we really go with getters on these over just making the final message fields public?

Reply:
we typically use `public final` field, I can change this as well

Reply:
Good with less verbosity in the samples I think, makes them easier to digest quickly.